blueprint/duktape/src-noline/duk_config.h

```diff
@@ -3739,4 +3739,15 @@ typedef struct duk_hthread duk_context;
 #error unsupported: byte order detection failed
 #endif  /* defined(DUK_USE_BYTEORDER) */
 
+
+#if JUCE_DEBUG
+//TODO: We need a better system for allowing custom/user configured duk_config headers
```

Comment:
@nick-thompson we should probably create an issue for this.

Reply:
💯 for sure. I'll do that now

Reply:
https://github.com/nick-thompson/blueprint/issues/62